pkg/schema/spec/definitions.yaml

```diff
@@ -372,3 +372,52 @@
       isArray: false
   requiredPrivileges:
     - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.Powertrain.CombustionEngine.Torque
+  conversions: 
+    - originalName: engineTorque
+      originalType: float64
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.Powertrain.Transmission.CurrentGear
+  conversions: 
+    - originalName: gearSelection
+      originalType: float64
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA 
+- vspecName: Vehicle.OBD.FuelPressure
+  conversions: 
+    - originalName: fuelTankPressure
+      originalType: float64
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA 
```

Comment:
Is this new?

Reply:
no those are from the last PR I did back 10/1.